### PR TITLE
[th2-3375-parameters] changed root event name, fixed parameters

### PR DESCRIPTION
**Pull request by Dmitriy-Yugay**

### CODE REVIEW

File: src/main/java/com/exactpro/th2/fix/client/Main.java [hunk 1]
```diff
@@ -224,8 +224,7 @@ public static void run(Settings settings, MessageRouter<MessageGroupBatch> messa
         });
 
         Event rootEvent = MessageRouterUtils.storeEvent(eventRouter, Event.start()
-                        .description("Root event")
-                        .name(boxName + " " + Instant.now())
+                        .name("Sessions events " + Instant.now())
                         .type("Microservice")
                         .bodyData(FixBeanUtil.getSessionTable(settings.getSessionSettings()))
                 , null);
```
Comment (cordwelt) [inline src/main/java/com/exactpro/th2/fix/client/Main.java:230]: > null

Didn't we decide to use root-event from common as a parent for this event?

Comment (Dmitriy-Yugay) [reply src/main/java/com/exactpro/th2/fix/client/Main.java:230]: we discussed this issue with @alexander-zhilov and decided not to do additional nesting for comfortability purposes

Comment (cordwelt) [reply src/main/java/com/exactpro/th2/fix/client/Main.java:230]: So we're okay with 2 root-events? =/

Comment (alexander-zhilov) [reply src/main/java/com/exactpro/th2/fix/client/Main.java:230]: We do not have options to remove or edit a root event generated by CommonFactory.
Also i think that one more event nesting is inconvenient.
Let it be a two root events as temporary solution, we add a new feature to modify CommonFactory root event in the future.

**Pull request merged**